carbon-design-system/ibm-security
# feat(a11y): add DAP and Axe matchers to Jest, add TrendingCard tests

## Affected issues

- Resolves #248 

## Proposed changes

- add `@ibm/aat` & `axe-core` for a11y testing
- add Jest matchers similar to Carbon's: https://github.com/carbon-design-system/carbon/tree/ac14f81df750dde81d125e6a2a036e476723b5b9/tasks/jest/matchers
- add a11y tests to `TrendingCard`

## Testing instructions

To check that the DAP and Axe tests are actually checking the `TrendingCard`, I suggest pulling down this branch and adding something critically invalid to the `TrendingCard`, like `role="blah"`, and then running `yarn test` to see if the DAP & Axe checkers throw an error.
